languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/style.xml

```diff
@@ -10188,6 +10188,30 @@ USA
         </rulegroup>
 
 
+        <rule id='CIENTÍFICO_DAR_CONFERIR' name="[Universitário][Científico] dar → conferir" type="style" tags="picky" tone_tags="academic" default='temp_off'>
```

Comment:
IMO this rule is solid enough not to be `picky`. Maybe it can be `is_goal_specific`. @susanaboatto what do you think?

Reply:
Yes, I think we can remove `picky` and see how the rule performs